pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -601,13 +573,13 @@ def extract_basic_trig_properties(trial_dict, trigs, slide_dict, seg_dict,
 # =============================================================================
```

Comment:
Remove `get_grb_time` if no longer needed?

Reply:
Yes, it's gone: this was a very dangerous function with a hard-coded +1s!